cli/ingress-controller/main.go

```diff
@@ -548,7 +548,7 @@ func main() {
 				logger.Fatalf("failed to load admission webhook certificate: %s", err)
 			}
 		}
-		tlsConfig := &tls.Config{
+		tlsConfig := &tls.Config{ //nolint:gosec
```

Comment:
TODO: need to follow up on this, potentially https://github.com/Kong/kubernetes-ingress-controller/milestone/12 would be a reasonable time to drop old TLS versions.

Reply:
I created https://github.com/Kong/kubernetes-ingress-controller/issues/1065 as a follow up for this.